doc/manual/src/release-notes/rl-next.md

```diff
@@ -21,3 +21,15 @@
 * Error traces have been reworked to provide detailed explanations and more
   accurate error locations. A short excerpt of the trace is now shown by
   default when an error occurs.
+
+* In light of the new `^` syntax for store paths allowing one to be fully explicit, stop special-casing whether a store path in an installable ends in `.drv`.
+
+  For example,
+  ```shell-session
+  $ nix path-info /nix/store/gzaflydcr6sb3567hap9q6srzx8ggdgg-glibc-2.33-78.drv`
+  ```
+  gives info about the derivation itself, while
+  ```shell-session
+  $ nix path-info /nix/store/gzaflydcr6sb3567hap9q6srzx8ggdgg-glibc-2.33-78.drv^*`
```

Comment:
The test case uses `^'*'`; should it be the same here?

Reply:
This is quoting for bash, but I want to say what the syntax is independent of which shell the user is using / which quoting is needed.

Reply:
```suggestion
  $ nix path-info /nix/store/gzaflydcr6sb3567hap9q6srzx8ggdgg-glibc-2.33-78.drv^*
```